### ISSUE TITLE
Unable to play a 360 video on Quest 3

### ISSUE THREAD

**Issue by mb2882** [RenderHeads/UnityPlugin-AVProVideo#1774]
### Please DO NOT LINK / ATTACH YOUR PROJECT FILES HERE

**Describe the issue**
Trying to get a video to play in Meta Quest 3 headset and I keep getting the following error in Unity (with auto profiler connected) "Autoconnected Player" [AVProVideo] Error: Loading failed.  File not found, codec not supported, video resolution too high or insufficient system resources.

This video is the NASA ISS Space Exploration 360 video.  It plays just fine with using Unity's video player component and using a standard Unity skybox shader.  Video also plays fine when running in the editor but I realize that's not on device so it's unclear why there is a resource error with the AVPro Android plugin when Unity is able to play it.  I've tried multiple different settings on the Android platform specific tab of the media player too.  

I tried Video API Exo player and Media player, both failed with the same error.  
I also tried enabling OES Rendering and updating the skybox shader to use AVPro's VR/InsideSphere Unlit Android OES Only
No luck with that so I also unchecked OES rendering and tried with a normal AVPro skybox shader VR/InsideSphere Unlit (stereo+fog) and couldn't get that to work either.

The original demo video that comes with AVPro sample scenes works just fine.  Can the plugin not handle high resolution 360 stereo videos at this time?

**Your Setup (please complete the following information):**
 - Unity version: 2022.3.14f1
 - AVPro Video version (number and edition (trial/core/ultra/enterprise)):  AVPro 2.9.2, plugin v2.8.5f1-trial, Android core version
 - Operating system version: Windows 11 22H2
 - Device model: Meta Quest 3 headset
 - Video specs (resolution, frame-rate, codec, file size): 5760x5760, 30fps, H264 mpeg4 AVC part 10, avc1, file size 381MB

**To Reproduce**
1. Open up the the AVPro Demo_360stereo Unity scene
2. Replace the video in the media player with the updated NASA video described above
3. Build APK, video won't start, just shows all black environment inside the headset

**Logs**
If applicable, add error logs to help explain your problem.
"Autoconnected Player" [AVProVideo] Error: Loading failed.  File not found, codec not supported, video resolution too high or insufficient system resources.

**Screenshots**

**Videos**
Video is located here on youtube: https://www.youtube.com/watch?v=Seh-oDCoORM

### Please DO NOT LINK / ATTACH YOUR PROJECT FILES HERE
Instead email the link to us unitysupport@renderheads.com

**Comment by mb2882** [comment]
Hey,
1. Yes, video plays fine through Unity's video player.  No need to re-encode.  Video player component, rendering to a render texture that is applied to a skybox material.  Following Unity's documentation on how to render a 360 video to a skybox
2. Attempted a re-encode with h265 and it still didn't play in headset 
[logcat.txt](https://github.com/RenderHeads/UnityPlugin-AVProVideo/files/14514652/logcat.txt)

3. Logcat attached when I attempted the run with the h265 encoded video.

**Comment by mb2882** [comment]
Got a couple other logcats for you.  Hopefully you can decipher better what's going on with these than I can.
Logcat_v02 = this is using another video I have, H264, also plays just fine with Unity's default video player, but doesn't play with the AVPro media player when in the headset. 
[logcat_video02.txt](https://github.com/RenderHeads/UnityPlugin-AVProVideo/files/14514908/logcat_video02.txt)

And this logcat is using the Cones-360stereo-2K30-H264.mp4 that comes with the demo materials.  This one works properly in the headset.  
[logcat_AVPro_video.txt](https://github.com/RenderHeads/UnityPlugin-AVProVideo/files/14514920/logcat_AVPro_video.txt)

**Comment by Chris-RH** [comment]
Are you streaming these or loading them from local storage?
The error messages imply that either the directory doesn't exist or it can't be accessed
Have you enabled Player Settings > Android > Write Permission > External (SDCard)?

**Comment by mb2882** [comment]
I've got the box checked to use the write permissions on Android. When I start the app up, it even asks on the headset if I want to allow access to local storage, which I say yes to.  But that still doesn't seem to get the videos to work.

I can try putting them in the same location as the AVPro examples. Maybe that will help.  I'll report back soon!

**Comment by Chris-RH** [comment]
any luck?

**Comment by mb2882** [comment]
Yep, it works if I put the video assets in the streamingAssets location where the other AVPro examples are located.  Anywhere else in the project and it seems it can't find the video.  This is good info for me so I can at least insure my videos are always in that location.

Thanks again for pinging me again about this! It had slipped off my radar for a bit.
